### ISSUE TITLE
"Report an Issue" throws error in dist

### ISSUE THREAD

**Issue by ungoldman** [hypermodules/hyperamp#304]
![screen shot 2018-02-22 at 3 43 09 pm](https://user-images.githubusercontent.com/427322/36570486-19cfe818-17e7-11e8-9fe9-aa191a0317b3.png)

Looks like `pkg.bugs.url` (from `package.json`) isn't there in the packaged build.

The link above it ("Learn More"), which links to `pkg.homepage`, works fine.

Offending line:

https://github.com/hypermodules/hyperamp/blob/e8c499488ada480742a3cd27f9e0c287a0dcb42d/main/menu.js#L44

**Comment by ungoldman** [comment]
Maybe `electron-builder` is stripping some properties from `package.json` during build?

**Comment by ungoldman** [comment]
Yep, here's the packaged `package.json`:

```json
{
  "name": "hyperamp",
  "description": "Indescribable music player",
  "version": "0.5.10",
  "author": "Hypermodules (https://github.com/hypermodules)",
  "browserify": {
    "transform": [
      "envify",
      "yo-yoify",
      "electron-renderify"
    ]
  },
  "dependencies": {
    "@bret/truthy": "^1.0.1",
    "bl": "^1.2.1",
    "choo": "^6.0.0",
    "choo-devtools": "^2.1.0",
    "classnames": "^2.2.5",
    "content-addressable-blob-store": "^4.5.1",
    "csjs-inject": "^1.0.1",
    "electron-context-menu": "^0.9.0",
    "electron-debug": "^1.4.0",
    "electron-default-menu": "^1.0.1",
    "electron-ipc-log": "^3.0.0",
    "electron-is-dev": "^0.3.0",
    "electron-log": "^2.2.12",
    "electron-store": "^1.1.0",
    "electron-updater": "2.20.1",
    "electron-window-state": "^4.0.2",
    "entypo": "^2.1.0",
    "existy": "^1.0.1",
    "file-url": "^2.0.2",
    "flush-write-stream": "^1.0.2",
    "folder-walker": "^3.0.0",
    "format-duration": "^1.0.0",
    "fy-shuffle": "^1.0.0",
    "global": "^4.3.2",
    "lodash.debounce": "^4.0.8",
    "lodash.get": "^4.4.2",
    "lodash.throttle": "^4.1.1",
    "mkdirp": "^0.5.1",
    "mousetrap": "^1.6.1",
    "music-metadata": "^0.9.2",
    "nanobus": "^4.1.0",
    "nanocomponent": "^6.0.0",
    "nanologger": "^1.3.0",
    "nanotick": "^1.1.6",
    "pump": "^3.0.0",
    "raven": "^2.2.1",
    "raven-js": "^3.19.1",
    "run-parallel": "^1.1.6",
    "through2-filter": "^2.0.0",
    "xtend": "^4.0.1"
  },
  "engines": {
    "npm": ">=5.0.0",
    "yarn": ">=1.0.0"
  },
  "homepage": "https://github.com/hypermodules/hyperamp",
  "license": "GPL-3.0",
  "main": "main/index.js",
  "productName": "Hyperamp",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/hypermodules/hyperamp.git"
  }
```

No `bugs` property.

So either we manually paste those links into the help menu, or we come up with some programmatic way to extract those from package.json at build time (maybe with browserify).

I'm okay with pasting as a quick fix. Thoughts @bcomnes?

**Comment by bcomnes** [comment]
Pasting is fine, but buhhh why would builder touch that?

**Comment by ungoldman** [comment]
Looks like this got changed in electron-builder in the three days ago:

https://github.com/electron-userland/electron-builder/commit/58933e03e4264934d3c7b0060af7b49b8584e79a

**Comment by ungoldman** [comment]
So looks like this has been resolved by a later version of electron-builder:

https://github.com/electron-userland/electron-builder/releases/tag/v20.0.7

But I know they move fast and are not afraid of introducing breaking changes... not sure what version we're currently using, will take a look and see if I can fix this now.

**Comment by ungoldman** [comment]
Fixed in #305. Release pending.
